docs/user/guides/technical-docs-seo-guide.rst

```diff
@@ -81,18 +85,25 @@ which we'll discuss shortly,
 the most important thing is making sure the spider can follow links on your site
 and get to all your pages.
 
-Avoid orphan pages ✅️
-~~~~~~~~~~~~~~~~~~~~~~
+Avoid unlinked pages ✅️
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+When building your documentation,
+you should ensure that pages aren't *unlinked* or *orphaned*,
+meaning that no other pages or navigation have a link to them.
 
-Sphinx calls pages that don't have links to them "orphans"
-and will throw a warning while building documentation that contains an orphan
-unless the warning is silenced with the :ref:`orphan directive <sphinx:metadata>`:
+Search engine crawlers will not discover pages that aren't referenced.
```

Comment:
```suggestion
Search engine crawlers will not discover pages that aren't linked from somewhere else on your site.
```

Reply:
This was a good example of using more words to add clarity :)

Reply:
Agreed.